qiskit/algorithms/aux_ops_evaluator.py

```diff
@@ -26,18 +26,18 @@
 from qiskit.providers import Backend
 from qiskit.quantum_info import Statevector
 from qiskit.utils import QuantumInstance
-from qiskit.utils.deprecation import deprecate_function
+from qiskit.utils.deprecation import deprecate_func
 
 from .list_or_dict import ListOrDict
 
 
-@deprecate_function(
-    "The eval_observables function has been superseded by the "
-    "qiskit.algorithms.observables_evaluator.estimate_observables function. "
-    "This function will be deprecated in a future release and subsequently "
-    "removed after that.",
-    category=PendingDeprecationWarning,
+@deprecate_func(
+    additional_msg=(
+        "Instead, use the function "
+        "``qiskit.algorithms.observables_evaluator.estimate_observables``."
```

Comment:
These can be Sphinx's references now :) We only want https://github.com/Qiskit/qiskit-terra/pull/9815 so that the runtime warning doesn't include noise like the `:ref:`.

But, if it's okay with you, I'd prefer to not spend more time re-updating all these references to use `:ref`, at least this week. Migrating the whole repo to use these new decorators took several hours, and fixing up https://github.com/Qiskit/qiskit-terra/pull/9532 and friends will take me some time too. I'm trying to balance this project w/ other docs priorities we have for March/early April. 

I'm happy to open a ticket to track this improvement, though, so that we don't forget we want to do it.